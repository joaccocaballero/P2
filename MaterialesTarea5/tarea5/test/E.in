filtradaOrdenada -1
filtradaOrdenada 6 0
insertarAlFinal 6 0.0
filtradaOrdenada -1
filtradaOrdenada 6 0
filtradaOrdenada 5 0
filtradaOrdenada -1
insertarAlFinal 4 0.0
insertarAlFinal 5 0.0
filtradaOrdenada -1
filtradaOrdenada 5 6 7 0
Fin