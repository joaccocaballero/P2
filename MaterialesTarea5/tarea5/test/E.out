1>
2>
3>Se insertó (6,0.00) al final de la cadena.
4>
5>(6,0.00)
6>
7>
8>Se insertó (4,0.00) al final de la cadena.
9>Se insertó (5,0.00) al final de la cadena.
10>
11>(5,0.00)(6,0.00)
12>Fin.
